www/lib/encryption/hooks.ts

```diff
@@ -0,0 +1,46 @@
+import { useSigner } from 'wagmi';
+import { useCallback } from 'react';
+import { createEncryptionKey } from './core';
+import { useEncryptionStore } from './store';
+import nacl from 'tweetnacl';
+
+const message = `Sign this message if you trust this application to access private information, such as names, addresses, and emails. It costs nothing to sign this message.`;
```

Comment:
can we call this `signingScope` to prevent it from being conflated with messages that are encrypted?

Reply:
yup :+1: